### PR TITLE
STORM-67 Provide API for spouts to know how many pending messages there are

### PR DESCRIPTION
**Pull request by bourneagain**
Changes made to expose the pending tuple count saved inside the RotatingMap [ 
https://github.com/apache/storm/blob/master/docs/documentation/Acking-framework-implementation.md#pending-tuples-and-the-rotatingmap ]

### CODE REVIEW

File: storm-core/src/jvm/storm/trident/spout/RichSpoutBatchTriggerer.java [hunk 1]
```diff
@@ -27,17 +27,12 @@
...
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Random;
-import java.util.Set;
 import storm.trident.topology.TridentBoltExecutor;
 import storm.trident.tuple.ConsList;
 import storm.trident.util.TridentUtils;
 
+import java.util.*;
```
Comment (knusbaum) [inline storm-core/src/jvm/storm/trident/spout/RichSpoutBatchTriggerer.java]: Let's not import java.util.*.

**Comment by bourneagain** [comment]
I am working on few more review comments presently and shall update as soon as its available for further review. Thanks!

**Comment by HeartSaVioR** [comment]
+1

**Comment by bourneagain** [comment]
Thanks @HeartSaVioR .We can have this merged to master whenever we feel appropriate.

**Pull request merged**